Gemfile

```diff
@@ -35,6 +35,8 @@ else
     end
 
     case ENV['RAILS_VERSION']
+    when /^[56]/, /^7.0/
+      gem 'concurrent-ruby', '1.3.4'
```

Comment:
Rails <= v7.0 have a bug where ActiveSupport calls logger without having included/required it. This inclusion was inherited from the `concurrent-ruby` gem which also removed their inclusion in a recent update (`v1.3.5`). So testing against Rails earlier than 7.1 requires us to pin to the `concurrent-ruby` version right before the change.

Having said that, Rails 7.0 is eol and going to stop receiving security updates in about 2 months. Should we have a follow-up PR to redo what we're checking with CircleCI?

Reply:
If pinning this here works for now let's just go with that and we can revisit it when we add support for Rails 8.